.github/workflows/CI.yml

```diff
@@ -30,6 +30,14 @@ jobs:
     - name: make check
       run: |
         ./util/buildRelease/smokeTest chpl
+         if [$? -ne 0]
+          then
+             echo "Encountered issue executing smokeTest"
+          exit 1
+          else
+             echo "smokeTest Succeeded"
+          fi
+
```

Comment:
I don't think this is needed or changes behavior. The CI job will already fail if `./util/buildRelease/smokeTest` (or any step) returns a non-zero exit code so we don't need to check exit code here. The current issue is that `smokeTest` doesn't return a non-zero exit code if the tabs or trailing whitespace checks fail for some cases. In particular it's success/failure criteria is the number of lines on `stdout`, but if there's a fatal error that goes to stderr and stdout will appear empty and it doesn't report an error.

So I think it's https://github.com/chapel-lang/chapel/blob/02b95d5d8d92b8ea27ffe663fefa40ee1156f249/util/buildRelease/smokeTest#L106-L123 that need to be updated to check status code.

Reply:
Done!